NOTICE.txt

```diff
@@ -10,30 +10,37 @@ This product includes software developed by Aurelius (http://thinkaurelius.com/)
  * Marko A. Rodriguez
  * Stephen Mallette
  * Pavel Yaskevich
+ * Others as listed in CONTRIBUTORS.txt
```

Comment:
https://github.com/JanusGraph/janusgraph/issues/203 is tracking the updates, thanks for filing. Point taken re: acknowledging the original Aurelius team, will phrase accordingly.